redux-logger/redux-logger.d.ts

```diff
@@ -8,13 +8,15 @@
 declare module 'redux-logger' {
 
   interface ReduxLoggerOptions {
+    actionTransformer?: (action: any) => any;
```

Comment:
Type "Object" in TS can not have properties besides defaults (toString, hasOwnProperty, etc). So in TypeScript-way action has type "any".